### PR TITLE
Improved comments by @rexalex

### PR DESCRIPTION
**Pull request by avoinea**
- Reply comment
- Better icons for users with no icon set
- Tree view of comments

comments

### CODE REVIEW

File: src/components/manage/Form/Form.jsx [hunk 1]
```diff
@@ -518,6 +524,24 @@ class Form extends Component {
     return id;
   }
 
+  /**
+   * will prevent event from triggering submit
```
Comment (tiberiuichim) [inline src/components/manage/Form/Form.jsx:528]: Comment should follow the style of the rest of the file.

**Review by tiberiuichim — APPROVED** [review]
Awesome! I've left a comment. Also, the Form API changes are worth mentioning in the changelog

**Comment by tisto** [comment]
Nice. Without having time to look at the code right now: how do we decide if we show threaded comments or flat ones? Do we take the p.a.registry setting from Plone classic into account here?

I am +1 on adding avatar images to the comments (which is the case in Plone classic if I am not mistaken). Though, we might want to do this in a separate PR.

**Comment by avoinea** [comment]
> Nice. Without having time to look at the code right now: how do we decide if we show threaded comments or flat ones? Do we take the p.a.registry setting from Plone classic into account here?

@tisto We do, but I don't remeber seeing settings about flat vs threaded comments.

> I am +1 on adding avatar images to the comments (which is the case in Plone classic if I am not mistaken). Though, we might want to do this in a separate PR.

Indeed, I think we should consider this checkbox from Site Setup > Discussion:

discussion

@rexalex ^

**Review by sneridagh — APPROVED** [review]

**Pull request merged**